### PR TITLE
Hide post-high school training questions if not relevant

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#2069

screen shot 2017-05-09 at 5 38 59 pm
__________________________________________________________________________________________________________
screen shot 2017-05-09 at 5 39 11 pm
__________________________________________________________________________________________________________
screen shot 2017-05-09 at 5 39 50 pm

### CODE REVIEW

File: src/js/edu-benefits/5490/config/form.js [hunk 1]
```diff
@@ -518,11 +518,21 @@ const formConfig = {
...
+                  const status = _.get('highSchool.status', form);
+                  return status === 'discontinued' || status === 'graduationExpected' || status === 'neverAttended';
+                }
+              }
             },
             postHighSchoolTrainings: _.merge(postHighSchoolTrainingsUi, {
               'ui:options': {
-                expandUnder: 'view:hasTrainings'
+                expandUnder: 'view:hasTrainings',
+                hideIf: form => {
```
Comment (jbalboni) [inline src/js/edu-benefits/5490/config/form.js]: Hmm, ideally you wouldn't need to do this. I'm going to add something about it to our expandUnder issue (https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2046)

File: test/edu-benefits/5490/config/educationHistory.unit.spec.jsx [hunk 2]
```diff
@@ -58,7 +58,7 @@ describe('Edu 5490 educationHistory', () => {
       }
     });
 
-    expect(formDOM.querySelectorAll('input,select').length).to.equal(12);
+    expect(formDOM.querySelectorAll('input,select').length).to.equal(10);
```
Comment (jbalboni) [inline test/edu-benefits/5490/config/educationHistory.unit.spec.jsx]: Can you add a test that verifies that when you choose `expected` for the high school status you get the diploma date and not the trainings question? Should look pretty similar to the `should render diploma date` test.

**Review by jbalboni — APPROVED** [review]
Looks good. I think you can merge once you add the test I mentioned.

File: test/edu-benefits/5490/config/educationHistory.unit.spec.jsx (hunk 2 above)
Comment (cehsu) [inline test/edu-benefits/5490/config/educationHistory.unit.spec.jsx]: Thanks for the feedback, implemented in commit 37b25a3.

File: src/js/edu-benefits/5490/config/form.js (hunk 1 above)
Comment (cehsu) [inline src/js/edu-benefits/5490/config/form.js]: Sounds good.

**Pull request merged**